### PR TITLE
Add more tones to supported list

### PR DESCRIPTION
**Pull request by jfsoul**
## What does this change?
Adds all of the remaining tones from the 5 minutes of logs I scanned this morning that we still don't "support". It turns out that most of these we do already support, and the ones we don't support so well (graphics, quizzes) will not be served by DCR anyway because they contain elements we don't support.

My process for checking whether there was a valid reason to leave these off the list was to check the first 5-10 articles of each tone for any problems that wouldn't be covered by another rule in ArticlePicker. For more unusual tones (resource) I also looked back further to see what other kinds of articles have used that tone in the past.

It will be interesting to see after this is merged whether there are still any tones we're missing. We may be close to the point where we no longer use an allow list and just allow every tone.

## Does this change need to be reproduced in dotcom-rendering ?

- [x] No
- [ ] Yes (please indicate your plans for DCR Implementation)

## Screenshots
N/A

## What is the value of this and can you measure success?
A few more couldRender points 🤞 

### Does this affect other platforms?

- [ ] AMP 
- [ ] Apps
- [ ] Other (please specify)

### Does this affect GLabs Paid Content Pages? Should it have support for Paid Content?

- [ ] No
- [ ] Yes (please give details)

### Does this change break ad-free?

- [ ] No
- [ ] It did, but tests caught it and I fixed it
- [ ] It did, but there was no test coverage so I added that then fixed it

### Does this change update the version of CAPI we're using?

- [ ] No, all the existing database files are just fine
- [ ] Yes, and I have [re-run all the tests locally and checked in all the updated data/database/xyz files](https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/15-updating-test-database.md)

### Accessibility test checklist

- [ ] [Tested with screen reader](https://accessibility.gutools.co.uk/testing/web/screen-readers/)
- [ ] [Navigable with keyboard](https://accessibility.gutools.co.uk/testing/web/keyboard-navigation/)
- [ ] [Colour contrast passed](https://accessibility.gutools.co.uk/testing/web/colour-contrast/)

### Tested

- [ ] Locally
- [ ] On CODE (optional)

### CODE REVIEW

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 1]
```diff
@@ -123,13 +123,17 @@ object ArticlePageChecks {
       "tone/albumreview",
       "tone/analysis",
       "tone/blog",
+      "tone/callout",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:126]: seem to all use formstack forms (which appear to be supported)

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 2]
```diff
@@ -123,13 +123,17 @@ object ArticlePageChecks {
...
+      "tone/childrens-user-reviews",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:127]: Nothing unusual about these apart from they don't use main media. Mostly old too.

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 3]
```diff
@@ -123,13 +123,17 @@ object ArticlePageChecks {
...
       "tone/comment",
       "tone/competitions",
       "tone/documentaries",
       "tone/editorials",
+      "tone/event-descriptions",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:132]: Nothing unusual here, not used much any more.

**Review by rcrphillips — APPROVED** [review]

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 4]
```diff
@@ -123,13 +123,17 @@ object ArticlePageChecks {
...
       "tone/explainers",
       "tone/extract",
       "tone/features",
+      "tone/graphics",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:136]: Not used often but usually contain interactive embeds which are not supported anyway.

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 5]
```diff
@@ -140,7 +144,10 @@ object ArticlePageChecks {
       "tone/performances",
       "tone/polls",
       "tone/profiles",
+      "tone/q-and-as",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:147]: Nothing unusual I could see

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 6]
```diff
@@ -140,7 +144,10 @@ object ArticlePageChecks {
...
+      "tone/quizzes",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:148]: Quiz atoms which are not supported - maybe I ought to check older ones for these?

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 7]
```diff
@@ -140,7 +144,10 @@ object ArticlePageChecks {
...
       "tone/recipes",
+      "tone/resource",
```
Comment (jfsoul) [inline article/app/services/dotcomponents/ArticlePicker.scala:150]: University guides and match previews were most of them. Look fine on DCR.

Comment (jfsoul) [reply article/app/services/dotcomponents/ArticlePicker.scala:150]: (apart from missing interactives)

**Review by shtukas — APPROVED** [review]

**Pull request merged**